### DISCUSSION TITLE
Error: Unable to locate conda environment 'tensorflow'  (in R studio)

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#289] [Q&A] [answered]
#LTSM
Halo, saya mendapatkan error ketika menggunakan function pada model LTSM for Time Series di R studio. Sebelumnya saya sudah menginstall modul tensorflow di python. Code saya seperti di bawah : 
```
use_condaenv("tensorflow")
```
Error saya :
```
Error in use_condaenv("tensorflow") : 
  Unable to locate conda environment 'tensorflow'.
```

**Comment by onlyphantom** [comment, marked as answer]
Hi @yohanespm77, `use_condaenv` function expects an environment name, **not the package / library name**. 

If you create multiple virtual environments and forgot what you name it, try `conda env list`. You should see a list of your virtual environments, pick the one that has `tensorflow` installed, and then call it with `use_condaenv("my_env_name")`

**Comment by NabiilahArdini** [comment]
Halo @yohanespm77, untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan close pada diskusi ini karena belum ada respon setelah 7 hari respon terakhir. Semoga dapat dipahami.
